docs/api/hazards.md

```diff
@@ -28,16 +28,29 @@ By "kidnap" we denote the action of manually lifting the robot and, eventually,
 The Create® 3 robot combines together different sensors data in order to determine when it's being kidnapped.
 A boolean status will be periodically published on the `kidnap_status` topic.
 
+### Reflective Infrared Sensors
 
-### The `ir_intensity` topic
+The Create 3 robot is equipped with 11 reflective IR sensors to detect hazards.
 
-Besides the aforementioned sensors, the Create® 3 robot is also equipped with [7 sets of IR emitters and receivers](/hw/mechanical/#ir-proximity-sensors) and it can use them to detect objects at close range.
+#### The `ir_intensity` topic
+
+[7 sets of IR emitters and receivers](/hw/mechanical/#ir-proximity-sensors) are available in the front bumper to detect objects at close range.
```

Comment:
Do we publish the actual range of those sensors anywhere? I'm genuinely curious to know.

Reply:
I don't know that we do. I believe the IR intensity is [0, 4096) and the cliff intensity is (-4095, 4096), but I need to verify this. Probably we should be clamping the cliff intensity at zero.